client/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { ButtonStyles, AuthenticationButton } from "./ButtonStyles";
+
+export default function Button() {
+  return <ButtonStyles>Search</ButtonStyles>;
+};
```

Comment:
We should change this file to something like this:
```
type Props = {
  content: string;
  name: string;
}

export default function Button({ content, name }: Props) {
  return <ButtonStyles id=`${name}-button-id`>{content}</ButtonStyles>;
};
```
Then just remove the AuthButton function, this should help us be more dynamic with this element to have it work with any type of name or functionality. Test it to see if it works. You will need to pass both the content and the name into this function.